manifests/n/Nozbe/NozbePersonal/3.20.1/Nozbe.NozbePersonal.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate
+
+PackageIdentifier: Nozbe.NozbePersonal
+PackageVersion: 3.20.1
+PackageLocale: en-US
+Publisher: Nozbe
+PackageName: Nozbe Personal

```

To match the ARP Table.
```suggestion
PackageName: Nozbe 3.20.1
```

---

There is also a product called Nozbe Teams, in addition to Nozbe Personal - so I will rename the package to "Nozbe Personal 3.20.1"